jdat_notebooks/ifu_optimal/requirements.txt

```diff
@@ -0,0 +1,12 @@
+notebook==6.1.4
+jupyter_client==5.3.5
+numpy==1.19.2
+scipy==1.5.2
+specutils==1.1.1
+git+https://github.com/glue-viz/glue.git
+git+https://github.com/glue-viz/glue-jupyter.git
+git+https://github.com/spacetelescope/jdaviz
+regions==0.4
+photutils=1.0.1
```

Comment:
There is one more "=" missing, should be `photutils==1.0.1`

Reply:
Flake8 checking complete.  All code is now PEP8 compliant.

Reply:
Fixed missing '=' in requirements and set explicit glue versions.